### PR TITLE
[TMVA-SOFIE] Fixed bugs in Slice SOFIE Operator, added slice gtests

### PR DESCRIPTION
**Pull request by joannapng**
- Fixed bugs in the parser for the Slice ONNX Operator, like adding code that handles cases where Steps/Axes attribute is missing.
- Added missing gtests for the Slice Operator

### CODE REVIEW

**Comment by lmoneta** [comment]
@phsft-bot build!

**Comment by lmoneta** [comment]
@phsft-bot build!

**Review by lmoneta — APPROVED** [review]
LGTM!
Thank you Ioanna for this fix.

File: tmva/sofie/inc/TMVA/ROperator_Slice.hxx [hunk 1]
```diff
@@ -97,13 +102,26 @@ public:
       // loop on the extra 2 or 3 or 4 inputs
       for (size_t i = 0; i < fNames.size(); ++i) {
         if (!fNames[i].empty()) { 
-         std::cout << " i " << i << " getting data for tensor " << fNames[i] << std::endl;
+         // std::cout << " i " << i << " getting data for tensor " << fNames[i] << std::endl;
```
Comment (sanjibansg) [inline tmva/sofie/inc/TMVA/ROperator_Slice.hxx:105]: we can maybe remove this line, since it is no longer required.

**Review by sanjibansg — APPROVED** [review]
LGTM!
Great work, thanks for the contribution. 
We can remove maybe the commented code since it is no longer required, and if needed in the future, we can retrieve from git history.

File: tmva/sofie/inc/TMVA/ROperator_Slice.hxx (hunk 1 above)
Comment (lmoneta) [reply tmva/sofie/inc/TMVA/ROperator_Slice.hxx:105]: I will remove it in a separate PR, since Joanna has left

**Pull request merged**